packages/component/src/Localization/ja-JP.js

```diff
@@ -41,14 +41,24 @@ function botSaidSomething(avatarInitials, text) {
   return `ボット${avatarInitials}は${text}と言いました`;
 }
 
+function downloadFileWithFileSize(downloadFileText, fileName, size) {
```

Comment:
I'm just gonna trust you on this file....

Reply:
lmao